stock_picking_update_responsible/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Stock Picking Update Responsible",
+    "summary": """
+        Update responsible when user print picking operations
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "17.0.1.0.0",
+    "category": "Stock",
+    "website": "https://github.com/solvosci/slv-stock",
+    "data": [
+        "security/security.xml",
```

Comment:
Better a more precise name:
```suggestion
        "security/stock_picking_update_responsible.xml",
```